go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/aws/amazon-cloudwatch-agent-test
 go 1.18
 
 require (
+	github.com/aws/aws-sdk-go v1.44.188
```

Comment:
Why do we need to introduce v1 of the SDK?

Reply:
Nice catch. We don't though, I mixed it up with another commit from another branch. Sorry for this.